bazel/scripts/filter_test_targets.sh

```diff
@@ -68,6 +68,7 @@ fi
 
 # Optional script argument, e.g. 'cc_test' or 'py_test', default is all test rules.
 BAZEL_FILTER_RULE=${1:-".*_test"}
+BAZEL_FILTER_TAG=${2:-".*"}
```

Comment:
do we have (or could we introduce) a use case where we have no rule filtering, but need a tag filtering? then we need input via flags.

Reply:
Currently there is no use case for that, but if it would become necessary, then you could just provide ".*" as the first parameter.